services/QuillLMS/app/mailers/user_mailer.rb

```diff
@@ -145,4 +145,13 @@ def ell_starter_diagnostic_info_email(name, email)
     mail from: "The Quill Team <hello@quill.org>", to: email, subject: "ELL Starter Diagnostic Next Steps"
   end
 
+  private def link_for_setting_password(role)
+    params = {
+      accountType: role,
+      adminFullName: @admin_name,
+      schoolName: @school_name
+    }
+    @set_password_link = "#{ENV['DEFAULT_URL']}/account/#{@user.token}/finish_set_up?#{params.to_query}"
+  end
```

Comment:
I would maybe just merge in the other branch and then point the head of this to that to avoid a merge conflict (since that one has to wait til the Segment events to ship anyway).